climada/entity/exposures/test/test_litpop.py

```diff
@@ -144,31 +141,61 @@ def test_reproject_input_data_downsample(self):
     def test_reproject_input_data_downsample_conserve_sum(self):
```

Comment:
Can you add at least one test with a custom target grid?

Reply:
Ideally, you can write a loop to test all the exposed methods (`from_countries` , `from_nightlight_intensity` etc ...)